app/mm-redux/client/client4.ts

```diff
@@ -265,6 +265,14 @@ export default class Client4 {
         return `${this.getPluginsRoute()}/${pluginId}`;
     }
 
+    getPluginMobileRoute(pluginId: string) {
+        return `${this.url}/plugins/${pluginId}/mobile`;
```

Comment:
We may want to consider the full path handled by the server - or even the plugin - how it currently does with interactive dialog requests to avoid clients (mobile/webapp) handling this differently.

Reply:
I have been making some changes to do that. Instead of a "trigger" word, we will be passing a requestURL, as you proposed on the Plugin Action document.